rq_dashboard/cli.py

```diff
@@ -162,6 +214,9 @@ def make_flask_app(config, username, password, url_prefix, compatibility_mode=Tr
 @click.option(
     "-j", "--json", is_flag=True, default=False, help="Enable JSONSerializer"
 )
+@click.option(
+    "--oidc-public-key", default=None, help="Public key for OIDC provider (needed if ouath2_proxy is used)"
```

Comment:
Verification key for oauth2 access_token verification (when oauth2 provider is used as middleware or reverse proxy. For example: oauth2_proxy+KeyCloak)